packages/app/contexts/ModalContext.tsx

```diff
@@ -16,6 +16,12 @@ export enum ModalId {
   CANCEL_STACK_PROCESSING = "cancel_stack_processing",
   CANCEL_STACK_SUCCESS = "cancel_stack_success",
   SUCCESS_STACK_TOAST = "success_stack_toast",
+  APPROVE_CONFIRM = "approve_confirm",
```

Comment:
this should have Stack on the name as well.

```suggestion
  STACK_APPROVE_CONFIRM = "stack_approve_confirm",
```